target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
```

Comment:
Should not commit `target`, this directory should be listed on `.gitignore`

Reply:
fixed